src/index.html

```diff
@@ -9,10 +9,44 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet is linked to a `.scss` file. Browsers cannot directly interpret SCSS files; they need to be compiled into CSS. Ensure that the SCSS file is compiled to a CSS file and link the resulting CSS file instead.